crates/ra_hir/src/resolve.rs

```diff
@@ -190,7 +190,7 @@ impl Resolver {
             .flatten()
     }
 
-    fn module(&self) -> Option<(&CrateDefMap, CrateModuleId)> {
+    pub(crate) fn module(&self) -> Option<(&CrateDefMap, CrateModuleId)> {
```

Comment:
as I wrote below, I think exposing a `krate` method would be nicer. this isn't really the interface I would want for a `module` method that's used outside of this module :)